gtk2_ardour/audio_time_axis.h

```diff
@@ -87,6 +87,9 @@ class AudioTimeAxisView : public RouteTimeAxisView
 	void show_existing_automation (bool apply_to_selection = false);
 	void hide_all_automation (bool apply_to_selection = false);
 
+ protected:
+	bool can_edit_name () const override;
```

Comment:
This is not C++98, but since the parent method is declared `virtual` "override" isn't needed.

This should also be AND modified with parent `RouteTimeAxisView::can_edit_name()`